tests/src/Functional/NewsIntegrationTest.php

```diff
@@ -73,24 +76,33 @@ public function testCreateNews() {
 
     // Create a News item.
     $this->drupalGet('node/add/oe_sc_news');
-    $page = $this->getSession()->getPage();
     $page->fillField('Title', 'Example title');
     $page->fillField('Content', 'Example Content');
     $page->fillField('Introduction', 'Example Introduction');
-    $page->fillField('Date', '2022-01-24');
     $media_name = $media_image->getName() . ' (' . $media_image->id() . ')';
     $page->fillField('Media item', $media_name);
     $page->fillField('Caption', 'Starter Image caption');
     $page->pressButton('Save');
 
-    // Assert media document has been created.
+    // Assert media document has been created and publication date
+    // is current date because it's not fulfilled.
```

Comment:
"fulfill" does not mean what you think it does.
```suggestion
    // Assert that news was created with the expected field values.
```
Put another comment further below about pub date.

Reply:
(the old comment was wrong, it talked about media document - or did I miss something?)